test/src/main/java/org/axonframework/test/aggregate/AggregateTestFixture.java

```diff
@@ -115,12 +159,17 @@ public AggregateTestFixture(Class<T> aggregateType) {
 
     @Override
     public FixtureConfiguration<T> registerAggregateFactory(AggregateFactory<T> aggregateFactory) {
+        MultiParameterResolverFactory parameterResolverFactory = MultiParameterResolverFactory.ordered(
+                new SimpleResourceParameterResolverFactory(resources),
+                ClasspathParameterResolverFactory.forClass(aggregateType)
+        );
+
         return registerRepository(new EventSourcingRepository<>(
-                aggregateFactory, eventStore,
-                MultiParameterResolverFactory.ordered(
-                        new SimpleResourceParameterResolverFactory(resources),
-                        ClasspathParameterResolverFactory.forClass(aggregateType)),
-                NoSnapshotTriggerDefinition.INSTANCE, getRepositoryProvider()));
+                aggregateFactory,
+                eventStore,
+                parameterResolverFactory,
+                NoSnapshotTriggerDefinition.INSTANCE, getRepositoryProvider()
```

Comment:
Since we're going for `parameter per line` strategy, I would recommend moving `getRepositoryProvider()` into a separate line.

Reply:
Ah missed this one, thanks for that.